icarusalg/Utilities/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
 
-cet_make(
+file(GLOB src_files *.cxx)
```

Comment:
OK for first pass: CMake best practice is to avoid `GLOB`bed library source lists wherever possible.